src/useInfiniteScroll.js

```diff
@@ -4,6 +4,7 @@ import useInterval from './useInterval';
 
 const WINDOW = 'window';
 const PARENT = 'parent';
+const SELF = 'self';
```

Comment:
We should add this to the type definition of `scrollContainer` in `index.d.ts`.
Otherwise, TypeScript developers can't use this value.

Reply:
sorry, i don’t use TypeScript. i can’t fix it. ^_^

Reply:
Ok, I can change type definitions after we try and merge this pr. No problem :)